components/library/giflib/patches/01-makefile.patch

```diff
@@ -0,0 +1,107 @@
+--- giflib-5.2.2/Makefile.~1~	2024-02-19 03:01:50.000000000 +0200
++++ giflib-5.2.2/Makefile	2025-11-13 21:38:55.835395949 +0200
+@@ -6,18 +6,17 @@
+ # of code space in the shared library.
+ 
+ #
+-OFLAGS = -O0 -g
+-OFLAGS  = -O2
+-CFLAGS  = -std=gnu99 -fPIC -Wall -Wno-format-truncation $(OFLAGS)
++
++CFLAGS  =  -fPIC -Wall
+ 
+ SHELL = /bin/sh
+ TAR = tar
+ INSTALL = install
+ 
+-PREFIX = /usr/local
++PREFIX = /usr
+ BINDIR = $(PREFIX)/bin
+ INCDIR = $(PREFIX)/include
+-LIBDIR = $(PREFIX)/lib
++LIBDIR = $(PREFIX)/lib/MACH64
+ MANDIR = $(PREFIX)/share/man
```

Comment:
\$\(MACH64\)

Reply:
And I tested $(MACH64) it's not propagated into the real build - so the need for that sed statement.